server/database/db_connection.js

```diff
@@ -0,0 +1,11 @@
+const { Pool } = require('pg')
+require('env2')('./config.env')
+
+const connectionString = process.env.DATABASE_URL
```

Comment:
This is good, can you make sure you add spaces between groups though to keep files tidy e.g. before module.exports